counters/count.py

```diff
@@ -6,3 +6,6 @@ def sub(x, y):
 
 def mul(x, y):
     return x * y
+
+def div(x, y):
```

Comment:
Change variable names to be more descriptive

Reply:
Fixed in 857c2f3